### PR TITLE
Add implemented phpBB version

### LINKED ISSUE #75: Feature - Provide released version for implemented ideas
It could be nice to be able to add the version of the first release containing the implementation of the idea

### PR DESCRIPTION
**Pull request by VSEphpbb**
Closes #75

### CODE REVIEW

File: styles/prosilver/template/idea_body.html [hunk 1]
```diff
@@ -84,6 +84,16 @@
 					{% endif %}
 				</dd>
 			{% endif %}
+			{% if IDEA_IMPLEMENTED or S_IS_MOD %}
+				<dt class="implementedtoggle idealabel"{% if IDEA_STATUS_ID != 3 %} style="display:none"{% endif %}>{{ lang('IMPLEMENTED_VERSION') ~ lang('COLON') }}</dt>
```
Comment (marc1706) [inline styles/prosilver/template/idea_body.html]: Any reason why you're using magic numbers in here?

**Comment by VSEphpbb** [comment]
Bump

File: language/en/common.php [hunk 2]
```diff
@@ -43,7 +43,9 @@
 	'IDEAS_TITLE'			=> 'phpBB Ideas',
 	'IDEAS_NOT_AVAILABLE'	=> 'Ideas is not available at this time.',
 	'IMPLEMENTED'           => 'Implemented',
+	'IMPLEMENTED_ERROR'		=> 'Mst be a valid phpBB version number.',
```
Comment (marc1706) [inline language/en/common.php]: typo: `Mst` must be `Must`

**Comment by VSEphpbb** [comment]
Updated

**Pull request merged**